schemas/context/pushdetail.schema.json

```diff
@@ -0,0 +1,54 @@
+{
+    "meta:license": [
+        "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+        "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+        "you may not use this file except in compliance with the License. You may obtain a copy",
+        "of the License at https://creativecommons.org/licenses/by/4.0/"
+    ],
+    "$id": "https://ns.adobe.com/xdm/context/pushdetail",
+    "$schema": "http://json-schema.org/draft-06/schema#",
+    "title": "Push notification detail",
+    "type": "object",
+    "meta:extensible": true,
+    "description": "Push notification detail required to send mobile push notification to a specific mobile application on a given device.",
+    "definitions": {
+        "pushdetail": {
+            "properties": {
+                "xdm:token": {
+                    "title": "Push Token",
+                    "type": "string",
+                    "description": "Valid push token for the given push platform."
+                },
+                "xdm:platform": {
+                    "title": "Push Platform",
+                    "type": "string",
+                    "description": "Push provider, e.g. apns or fcm"
+                },
+                "xdm:blacklisted": {
+                    "title": "Blacklist Flag",
+                    "type": "boolean",
+                    "description": "Indicates whether the push token is blacklisted."
+                },
+                "xdm:appID": {
+                    "title": "Application Identifier",
+                    "type": "string",
+                    "description": "Unique Identifier for the mobile application."
+                },
+                "xdm:ECID": {
```

Comment:
ECID is the namespace value for the given IdentiyMap. Can you rename the property definition to be more generic  and call it as pushNotificationID

Reply:
is tokenID ok? conceptually speaking this is the ECID the push token was associated with in the originating profile pre-stitching.

Reply:
Will this notification always be based on ECID? today/ in the near future? I would have modelled it via an object with (Key and Value as a paired object) with 'ECID' being the Key and value of ECID being the value. That way it becomes generic.

Reply:
Today yes, on mobile we always create a new ecid on install and send in the profile event to platform with that ecid as the primary identity. For maximum flexibility what you suggested make sense, in case we move to some other id in the future, I'm not sure how to encode what you suggested properly in the schema, can you point to an example I can follow?

Reply:
I looked around a bit further and it looks like https://ns.adobe.com/xdm/context/profileStitchIdentity would do the job here, will this work?